### PR TITLE
[CI:DOCS] Apiv2

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

File: pkg/serviceapi/handler_images.go [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+	)
+	quiet := false
+	if len(r.Form.Get("quiet")) > 0 {
+		quiet, err = strconv.ParseBool(r.Form.Get("quiet"))
+		if err != nil {
+			Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "bad value for quiet"))
+			return
+		}
+	}
+	f, err := ioutil.TempFile("", "api_load.tar")
```
Comment (AkihiroSuda) [inline pkg/serviceapi/handler_images.go]: defer remove f?

File: pkg/serviceapi/handler_info.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+func getGraphStatus(storeInfo map[string]interface{}) [][2]string {
+	var graphStatus [][2]string
+	for k, v := range storeInfo["GraphStatus"].(map[string]string) {
+		graphStatus = append(graphStatus, [2]string{k, v})
+	}
+	return graphStatus
+}
+
+func getSecOpts(sysInfo *sysinfo.SysInfo) []string {
```
Comment (AkihiroSuda) [inline pkg/serviceapi/handler_info.go]: add rootless

File: pkg/serviceapi/handler_ping.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+func pingHEAD(w http.ResponseWriter, _ *http.Request) {
+	setHeaders(w)
+	fmt.Fprintln(w, "")
+}
+
+func setHeaders(w http.ResponseWriter) {
+	w.WriteHeader(http.StatusOK)
+	w.Header().Set("API-Version", DefaultApiVersion)
+	w.Header().Set("BuildKit-Version", "")
```
Comment (AkihiroSuda) [inline pkg/serviceapi/handler_ping.go]: unset?

File: pkg/serviceapi/handler_version.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+	infoData, err := s.Runtime.Info()
+	if err != nil {
+		Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrapf(err, "Failed to obtain system memory info"))
+		return
+	}
+	hostInfo := infoData[0].Data
+
+	components := []docker.ComponentVersion{{
+		Name:    "Podman Engine",
```
Comment (AkihiroSuda) [inline pkg/serviceapi/handler_version.go]: Do you need "Engine"?

File: pkg/serviceapi/handler_info.go (hunk 2 above)
Comment (baude) [reply pkg/serviceapi/handler_info.go]: no review is needed right now

**Comment by mheon** [comment]
@AkihiroSuda We're just committing to a branch right now to aid collaboration - final review when we land into master, probably next week

**Comment by baude** [comment]
> @AkihiroSuda We're just committing to a branch right now to aid collaboration - final review when we land into master, probably next week

also we would gladly take PRs on this if you want to contribute to this branch.

**Pull request merged**